code/code/task/task_picklock.cc

```diff
@@ -1,144 +1,122 @@
-//////////////////////////////////////////////////////////////////////////
-//
-//      SneezyMUD 4.0 - All rights reserved, SneezyMUD Coding Team
-//      "task.cc" - All functions related to tasks that keep mobs/PCs busy
-//
-//////////////////////////////////////////////////////////////////////////
-
 #include "extern.h"
 #include "comm.h"
 #include "room.h"
 #include "obj_tool.h"
 #include "being.h"
 
-void TThing::pickPulse(TBeing *ch)
-{
-  ch->sendTo("Hey, where'd your lockpick go?!?\n\r");
-  ch->stopTask();
-}
-
-void TTool::pickPulse(TBeing *ch)
+void pick_pulse(TBeing *ch, TThing *pick)
```

Comment:
namespace {}

Reply:
yeah, today I read something reminding me about throwing static on these kinds of things: I'm using that instead of a namespace as it's shorter. it used to be deprecated vs anon namespaces but it's back to a-ok again. added.